js/controller.js

```diff
@@ -245,7 +253,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       // Setting the tabindex will let some screen readers recognize this element as a group
       // identified with the ARIA label above. We set it to -1 in order to prevent actual keyboard focus
       this.state.mainVideoInnerWrapper.attr('tabindex', '-1');
-      this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>");
+
+      var $ooPlayerSkin = $('.oo-player-skin');
+      !$ooPlayerSkin.length && this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>");
```

Comment:
Please change this to an if statement.  LIke my other comment below, we don't call functions that change the value of variables within checks like this to allow for easier debugging.